Makefile

```diff
@@ -52,18 +52,20 @@ cross:
 	GOOS=windows $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_WINDOWS_CROSS)"
 	GOOS=darwin $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_DARWIN_CROSS)"
 
-tools: tools.timestamp
+tools: .install.gitvalidation .install.golangci-lint
 
-tools.timestamp: Makefile
-	@GO111MODULE="off" go get -u $(BUILDFLAGS) golang.org/x/lint/golint
```

Comment:
So, this removes the installation of `golint`? Would be nice to mention it in the description and commit message.

Reply:
Sure can add :+1: